addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -77,7 +77,6 @@
     'SM': 9951,
     'TR': 9952,
     'VA': 9953,
-    'SE': 9955,
```

Comment:
You need to replace the old number by the new one, I believe '0007' ?
Also please retarget 15.0.

Reply:
yes you are right, it is 0007